src/adapter/src/catalog/open.rs

```diff
@@ -266,13 +270,11 @@ impl Catalog {
             // This time is usually the current system time, but with protection
             // against backwards time jumps, even across restarts.
             let boot_ts = {
-                let previous_ts = txn
-                    .get_timestamp(&Timeline::EpochMilliseconds)
-                    .expect("missing EpochMilliseconds timeline");
```

Comment:
Probably not great that we're doing this right now in prod even though this `previous_ts` is meaningless because we used the shared ts oracle.

I'm not entirely sure what the consequences are though.